application/dash_application/known_pages_router.py

```diff
@@ -51,8 +51,8 @@ def menu_item(label, link):
 layout = html.Div([
     core.Location(id='url', refresh=False),
     navbar(default_title),
-    core.Loading(id='page-content'),
-    html.Footer(id='footer', children=[html.Hr(), "Dash version {0} | Data version ".format(version), html.Span(id='data-version')])
+    core.Loading(id='page-content', type='dot'),
+    html.Footer(id='footer', children=[html.Hr(), "Dash version {0}".format(version)])
```

Comment:
Maybe a bad rebase -- could the footer code from develop please be retained?

Reply:
Should be good now